packages/blade/src/utils/makeAccessible/accessibilityMapNative.ts

```diff
@@ -0,0 +1,99 @@
+import type { AccessibilityMap } from './makeAccessible.d';
+
+export const accessibilityValue = {
```

Comment:
no explicity type for this?

Reply:
Not needed we are already adding explicit type in the map object and spreading this object onto it.